MvvmCross/Core/Binding/Bindings/Target/Construction/MvxTargetBindingFactoryRegistry.cs

```diff
@@ -116,10 +116,16 @@ private IMvxPluginTargetBindingFactory FindSpecificFactory(Type type, string nam
             {
                 return factory;
             }
+            var implementedInterfaces = type.GetTypeInfo().ImplementedInterfaces;
+            foreach (var implementedInterface in implementedInterfaces)
+            {
+                factory = this.FindSpecificFactory(implementedInterface, name);
+                if (factory != null) return factory;
+            }
```

Comment:
It depends on the usage.

If binding to derived classes 'll happen more often than binding to interfaces, yes.

Since we have MvxViewClickBinding (and others), you are right, we should move that.